src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor

```diff
@@ -34,35 +34,11 @@ else if (Resource.TryGetExecutablePath(out var executablePath) && Resource.TryGe
 }
 else if (Resource.TryGetContainerImage(out var containerImage) && Resource.TryGetContainerPorts(out var ports))
 {
-    var portsString = string.Join("; ", ports);
     <GridValue Value="@containerImage"
                EnableHighlighting="true"
                HighlightText="@FilterText"
                PreCopyToolTip="@Loc[nameof(Columns.SourceColumnSourceCopyContainerToClipboard)]"
-               ToolTip="@containerImage">
-        <ContentAfterValue>
-           @if (ports.Length > 0)
-           {
-               var title = string.Format(
-                   Loc[ports.Length == 1
-                       ? nameof(Columns.SourceColumnDisplayPort)
-                       : nameof(Columns.SourceColumnDisplayPorts)],
-                   string.Join(",", ports));
-                   <span class="subtext" aria-label="@title">@title</span>
-           }
-        </ContentAfterValue>
-    </GridValue>
-    <FluentStack Orientation="Orientation.Horizontal">
-        @if (Resource.TryGetContainerCommand(out var command))
-        {
-            <div class="ellipsis-overflow" title="@Loc[nameof(Columns.SourceColumnDisplayContainerCommandTitle)]">@Loc[nameof(Columns.SourceColumnDisplayContainerCommand), command]</div>
-        }
-        @if (Resource.TryGetContainerArgs(out var args))
-        {
-            var argsString = string.Join(" ", args);
-            <div class="ellipsis-overflow" title="@Loc[nameof(Columns.SourceColumnDisplayContainerArgsTitle)]">@argsString</div>
-        }
```

Comment:
Can either of `TryGetContainerCommand` or `TryGetContainerArgs` be removed? I don't see any new usages in the code here.